ntc_templates/templates/huawei_vrp_display_bgp_peer.textfsm

```diff
@@ -0,0 +1,20 @@
+Value PEER ([\w\.:]+)
+Value V (\d+)
```

Comment:
@brian-s-cao
I presume that this `V` stands for the BGP Version, right?

Does this add value or is it worth capturing?

Reply:
Looks like V for version has been removed.
I'll resolve this sub-conversation section.